docs/pages/router/reference/static-rendering.mdx

```diff
@@ -158,7 +157,7 @@ Now the dynamic child routes will be invoked twice, once with `{ id: 'one' }` an
 
 ```tsx app/[id]/[comment].tsx
 export async function generateStaticParams(params: {
-  id: 'one' | 'two',
+  id: 'one' | 'two';
```

Comment:
```suggestion
  id: 'one' | 'two',
```

Might be a Prettier change, but it seems incorrect (since it's params object, not its types definition).